app/brave_generated_resources.grd

```diff
@@ -222,6 +222,9 @@ By installing this extension, you are agreeing to the Google Widevine Terms of U
       <message name="IDS_SETTINGS_HIDE_BRAVE_REWARDS_BUTTON_DESC" desc="The description for settings switch controlling the visibility of Brave Rewards button">
         Hides the Brave Rewards button in the location bar when Brave Rewards is not enabled
       </message>
+      <message name="IDS_SETTINGS_ALWAYS_SHOW_BOOKMARK_BAR_ON_NTP" desc="The label for settings switch controlling the visibility of bookmarks bar on NTP">
+        Always show bookmarks bar on NTP
```

Comment:
Is this a user-visible string? If so, I don't think the average user would understand what NTP means - probably better to spell it out instead of using an acronym.

Reply:
Agreed. @rebron Can you check this sentence?

Reply:
Can you have it read: "Always show bookmarks bar on New Tab page."